tensorboard/components/tf_feature_flags/tf-feature-flags-polymer.ts

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {customElement} from '@polymer/decorators';
+import {PolymerElement} from '@polymer/polymer';
+import * as tf_feature_flags from './feature-flags';
+
+/**
+ * This Polymer component allows the underlying API to be accessible from
+ * Angular TensorBoard by exposing otherwise mangled smybols.
+ */
+@customElement('tf-feature-flags')
+class TfFeatureFlags extends PolymerElement {
+  override _template = null;
+  tf_feature_flags = tf_feature_flags;
```

Comment:
Again, sorry, honestly I don't know. I'm following the pattern established for similar files, like:
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/components/tf_globals/globals-polymer.ts

Reply:
This might look confusing but syntactically it's not assigning a variable to itself; it's defining a property on the element and assigning its value to be the imported module from `import * as tf_feature_flags from './feature-flags';`

You can see how this mechanism works a bit better with the existing tf_globals code:

1) We create a `tf-globals` custom element that basically just holds a `tf_globals` property that represents the symbols from the Polymer static utilities module: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/components/tf_globals/globals-polymer.ts

2) We define a type for this property so that the TypeScript code on the Angular side can access it in a typesafe manner: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/tb_polymer_interop_types.ts;l=30-32;drc=046667d00df2316eefcda9a4ef058e83221bad77

3) In the Angular code where we actually need to call the `tf_globals` static utilities, we create a custom element, then access its `tf_globals` property: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/deeplink/hash.ts;l=30-35;drc=367839f30de3968eff5fecb8b38e1d5782434b6b